app/models/search/pseud_indexer.rb

```diff
@@ -25,7 +36,58 @@ def document(object)
     object.as_json(
       root: false,
       only: [:id, :user_id, :name, :description, :created_at],
-      methods: [:user_login]
-    )
+      methods: [
+        :user_login,
+        :byline,
+        :collection_ids
```

Comment:
`collection_ids` is for people search inside collections?

Reply:
Yes, it might not be very useful, but it still exists, so I thought we should keep it in the index.